qtclient/requestmicpermission.mm

```diff
@@ -0,0 +1,33 @@
+#import "requestmicpermission.h"
```

Comment:
Please add a GPLv2+ compatible license to source files you create. The easiest option is to copy the license header used by other files and adjust the copyright ownership to your name.